Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -164,6 +201,11 @@ const StatusBar = React.createClass({
       StatusBarManager.setNetworkActivityIndicatorVisible(visible);
     },
 
+    /**
+     * Set the background color for the status bar
+     * @param {string} color Background color.
+     * @param {boolean=} animated Animate the style change.
```

Comment:
Would there be any way to parse the type from the Flow type definitions below?

That way we could write:

    /**
     * Set the background color for the status bar
     * @param color Background color.
     * @param animated Animate the style change.
     */

- It would be less to type for people working on new API
- There would be no risk the docs and code go out of sync. But we have that risk anyway - people can change param names and not update the docs for example, I hope people won't do that :)

I think I already asked that before, sorry about asking here again. I'm fine with the way it's done in this PR too but if it's feasible to parse the Flow type definitions then would love it :)